curriculum/challenges/english/08-coding-interview-prep/data-structures/create-a-set-class.english.md

```diff
@@ -6,19 +6,26 @@ challengeType: 1
 
 ## Description
 <section id='description'>
```

Comment:
Just some minor suggestions for this section. The word set was capitalized a few times where it shouldn't be, some spacing, and I tried to simplify the three objectives of the challenge a bit:

```
<section id='description'>
In this exercise we are going to create a class named <code>Set</code> to emulate a data structure called a "set". A set is like an array, but it cannot contain duplicate values. The typical use for a set is to simply check for the presence of an item. This can be implemented with an object, for instance:

<blockquote>var set = new Object();<br>set.foo = true;<br>// See if foo exists in our set:<br>console.log(set.foo) // true</blockquote>

First, we will create a method adds a value to our set collection as long as the value does not already exist in the set.

Then, we will create a remove method that removes a value from the set collection if it already exists.

And finally, we will create a size method that returns the number of elements inside the set collection.
</section>
```

Reply:
The changes you suggested do in fact make it simpler to understand. Thank you! I'll update the PR.